.github/ISSUE_TEMPLATE/create_release_branch.md

```diff
@@ -0,0 +1,157 @@
+  - `git clone git@github.com:canonical/rawfile-localpv.git ~/tmp/release-1.xx`
+  - `pushd ~/tmp/release-1.xx`
+  - `git switch main`
+  - `git pull`
+  - `git checkout -b release-1.xx`
+  - `git push origin release-1.xx`
+  - `popd`
+  - `rm -rf ~/tmp/release-1.xx`
+- [ ] **Reviewer**: Ensure `release-1.xx` branch is based on latest changes on `main` at the time of the release cut.
+- [ ] **Reviewer**: Ensure `release-1.xx-strict` branch is based on latest changes on `autoupdate/strict` at the time of the release cut.
+- [ ] **Owner**: Create PR to initialize `release-1.xx` branch:
+  - [ ] Update `KUBE_TRACK` to `1.xx` in [/build-scripts/components/kubernetes/version.sh][]
+  - [ ] Update `master` to `1.xx` in [/build-scripts/components/k8s-dqlite/version.sh][]
+  - [ ] Update `"main"` to `"release-1.xx"` in [/build-scripts/hack/generate-sbom.py][]
+  - [ ] `git commit -m 'Release 1.xx'`
+  - [ ] Create PR with the changes and request review from **Reviewer**. Make sure to update the issue `Information` section with a link to the PR.
+- [ ] **Reviewer**: Review and merge PR to initialize branch.
+- [ ] **Reviewer**: On merge, confirm [Auto-update strict branch] action runs to completion
+- [ ] **Owner**: Create launchpad builders for `release-1.xx`
+  - [ ] Go to [lp:k8s][] and do **Import now** to pick up all latest changes.
+  - [ ] Under **Branches**, select `release-1.xx`, then **Create snap package**
+  - [ ] Set **Snap recipe name** to `k8s-snap-1.xx`
+  - [ ] Set **Owner** to `Canonical Kubernetes (containers)`
+  - [ ] Set **The project that this Snap is associated with** to `k8s`
+  - [ ] Set **Series** to Infer from snapcraft.yaml
+  - [ ] Set **Processors** to `AMD x86-64 (amd64)` and `ARM ARMv8 (arm64)`
+  - [ ] Enable **Automatically upload to store**
+  - [ ] Set **Registered store name** to `k8s`
+  - [ ] In **Store Channels**, set **Track** to `1.xx-classic` and **Risk** to `edge`. Leave **Branch** empty
+  - [ ] Click **Create snap package** at the bottom of the page.
+- [ ] **Owner**: Create launchpad builders for `release-1.xx-strict`
+  - [ ] Return to [lp:k8s][].
+  - [ ] Under **Branches**, select `release-1.xx-strict`, then **Create snap package**
+  - [ ] Set **Snap recipe name** to `k8s-snap-1.xx-strict`
+  - [ ] Set **Owner** to `Canonical Kubernetes (containers)`
+  - [ ] Set **The project that this Snap is associated with** to `k8s`
+  - [ ] Set **Series** to Infer from snapcraft.yaml
+  - [ ] Set **Processors** to `AMD x86-64 (amd64)` and `ARM ARMv8 (arm64)`
+  - [ ] Enable **Automatically upload to store**
+  - [ ] Set **Registered store name** to `k8s`
+  - [ ] In **Store Channels**, set **Track** to `1.xx` and **Risk** to `edge`. Leave **Branch** empty
+  - [ ] Click **Create snap package** at the bottom of the page.
+- [ ] **Reviewer**: Ensure snap recipes are created in [lp:k8s/+snaps][]
+  - look for `k8s-snap-1.xx`
+  - look for `k8s-snap-1.xx-strict`
+
+#### After release
```

Comment:
In microk8s we set the new release as the snaps new default as part of the release process. We should add this here as well. WDYT?
https://snapcraft.io/k8s/releases

Reply:
Actually, we usually wait a couple of weeks before switching the default snap track from `1.{xx-1}/stable` to `1.xx/stable`. But it might still be useful to have a section saying:

"- Observe CI and nightly runs for the new branch, once you are confident, switch the default track"